subsys/net/ip/Kconfig.debug

```diff
@@ -91,7 +91,16 @@ config NET_DEBUG_NET_PKT
 	default n
 	select NET_BUF_POOL_USAGE
 	help
-	Enables printing of network packet and buffer allocations and frees.
+	Enables collection of network packet and buffer allocations and frees.
+
+config NET_DEBUG_NET_PKT_ALL
+	bool "Debug network packet and buffer individual allocation"
```

Comment:
I'm no sure that from title alone it's clear what this option does vs NET_DEBUG_NET_PKT. Perhaps it can be clarified.